proj/src/reducers/time-tracking/timeTrackingReducer.js

```diff
@@ -1,6 +1,8 @@
-import { handleActions, combineActions } from 'redux-actions';
+import { handleActions } from 'redux-actions';
 import { set } from 'lodash/fp'
 
+import timeSet from './timeSetReducer'
```

Comment:
utils functions should sit under utils directory not in the reducer. the only case i can think of that this is ok is when the function sits in the reducer and serves that reducer alone